### PR TITLE
feat(deployment): Add quality updates to terraform deployment

### PR DESCRIPTION
**Pull request by felipecosta09**
# Add quality updates to terraform deployment

## Change Summary

- Update AWS provider to version `4.27.0`;
- Implement better file structure;
- Add all available variables to the template;
- Add a description to all variables;

## PR Checklist

- [X] I've read and followed the [Contributing Guide](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/.github/CONTRIBUTING.md).
- Documents/Readmes
  - [ ] Updated accordingly
  - [X] Not required
- Plugins that have versioning
  - [ ] Version bumped and follows [Semantic Versioning](https://semver.org/)
  - [X] Not required

### CODE REVIEW

File: deployment/aws-terraform/all-in-one/main.tf [hunk 1]
```diff
@@ -1,48 +0,0 @@
-terraform {
```
Comment (kwan-su) [inline deployment/aws-terraform/all-in-one/main.tf:1]: Why we remove main.tf? 
Please follow terraform standard structure https://www.terraform.io/language/modules/develop/structure

Comment (felipecosta09) [reply deployment/aws-terraform/all-in-one/main.tf:1]: Done!

**Review by kwan-su — APPROVED** [review]

**Comment by jack-c-tang** [comment]
Hi @felipecosta09 , would you please squash the commits to one?

**Comment by felipecosta09** [comment]
> Hi @felipecosta09 , would you please squash the commits to one?

It's done

**Review by jack-c-tang — APPROVED** [review]

**Pull request merged**